interfaceGráfica.py

```diff
@@ -1,5 +1,6 @@
 from tkinter import *
 from tkinter import ttk
+from tkinter import messagebox
 
 class Application:
```

Comment:
nao misturar a nomeclatura, se for ter coisas em português colocar tudo em português